src/index.html

```diff
@@ -7,12 +7,58 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img"></div>
+      <div class="card__details">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="card__description">Product code: 195434</h3>
+      </div>
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__reviews-text">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-title">Price:</p>
+        <p class="card__price-price">$2,199</p>
+      </div>
+      <a
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="card__price">
        <p class="card__price-title">Price:</p>
        <p class="card__price-price">$2,199</p>
      </div>
      
      <a
```